packages/react/src/components/SlashMenu/SlashMenuPositioner.tsx

```diff
@@ -33,65 +38,65 @@ export const SlashMenuPositioner = <
 
   const referencePos = useRef<DOMRect>();
 
+  const { refs, update, context, floatingStyles } = useFloating({
+    open: show,
+    placement: "bottom-start",
+    middleware: [
+      offset(10),
+      size({
+        apply({ availableHeight, elements }) {
+          Object.assign(elements.floating.style, {
+            // Minimum acceptable height is 200px.
+            // `flip` will then take over.
+            height: `${Math.max(200, availableHeight - 10)}px`,
+          });
+        },
+      }),
+      flip({
+        fallbackStrategy: "initialPlacement",
```

Comment:
Nice you were able to fix all this. maybe add an extra comment to explain the scenarios / why `initialPlacement` is needed